cmd/solana/main.go

```diff
@@ -0,0 +1,373 @@
+package main
```

Comment:
What is this cmd package used for?
Looks like it was used for testing? Can we consider removing it?
Otherwise there are lot of clean up to do in this file

Reply:
removed